src/core/actions/version.js

```diff
@@ -0,0 +1,29 @@
+import {
+  VERSION_GET,
+  VERSION_LOADED,
+  VERSION_FAILED,
+} from 'core/constants';
+
+
+export function versionGet({ slug, versionID }) {
+  return {
+    type: VERSION_GET,
+    payload: { slug, versionID },
+  };
+}
+
+export function versionLoad({ entities }) {
+  return {
+    type: VERSION_LOADED,
+    payload: {
+      result: entities ? entities.versions : null,
```

Comment:
Why should `entities` be null? The ternary operator checks for a *falsy* value. The `payload.result` would be `null`.